playbooks/roles/shadowsocks/templates/instructions.md.j2

```diff
@@ -64,6 +64,17 @@ For reference, this is the configuration specified:
         Password: {{ shadowsocks_password.stdout }}
         Encryption Method: {{ shadowsocks_encryption_method }}
 
+For users behind stricter surveillance (esp. experiencing QOS throttling in PRC), further configuration of shadowsocks along with simple-obfs plugin can be carried out via following configuration:
+
+        Server: {{ streisand_ipv4_address }}
+        Port: {{ shadowsocks_server_port }}
+        Password: {{ shadowsocks_password.stdout }}
+        Encryption Method: {{ shadowsocks_encryption_method }}
+        Plugin: {{ shadowsocks_obfs_plugin }}
+        Plugin_Options: {{ shadowsocks_obfs_client_plugin_opts }}
+
+Note: www.xxx.com should be replaced by a known host name like, www.cloudfront.com or www.baidu.com etc.
```

Comment:
See my earlier question about this config setting. Should it be replaced server-side? If so then I think this will be problematic. Ideally we can come up with a solution that can be used without manual configuration file editing on the Streisand instance.